core/src/main/scala/tofu/Errors.scala

```diff
@@ -8,12 +8,18 @@ import tofu.internal.{CachedMatcher, DataEffectComp}
 import tofu.lift.Lift
 import tofu.optics.PUpcast.GenericSubtypeImpl
 import tofu.optics.{Downcast, Subset, Upcast}
-
-trait Raise[F[_], E] extends Raise.ContravariantRaise[F, E] {
+import scala.annotation.implicitNotFound
+
+/**
+  * Having this means a possibity of raising `E` inside type `F`
```

Comment:
Why not simplifying the sentence?
```suggestion
  * Allows to raise `E` inside type `F`.
```

Reply:
Same for other similar comments below.

Reply:
Can merge all of them, if you do multisuggestion